test/unit/ui/control/fullscreen.test.js

```diff
@@ -27,3 +27,18 @@ test('FullscreenControl does not appears when fullscreen is not enabled', (t) =>
     t.equal(consoleWarn.getCall(0).args[0], 'This device does not support fullscreen mode.');
     t.end();
 });
+
+test('FullscreenControl makes optional container element full screen', (t) => {
+    window.document.fullscreenEnabled = true;
+
+    const map = createMap(t);
+    const fullscreen = new FullscreenControl({container: window.document.querySelector('body')});
+    map.addControl(fullscreen);
+    const control = map._controls.find((ctrl) => {
+        return ctrl.hasOwnProperty('_fullscreen');
+    });
+    control._onClickFullscreen();
```

Comment:
maybe set a spy on `body.requestFullscreen` and then check `t.calledOnce` after this?

Reply:
I might be missing something but it doesn't seem like JSDOM supports the Full Screen API. I don't see any documentation or code that mentions it and `window.document.requestFullScreen` and `body.requestFullScreen` are both `undefined` (along with their prefixed alternatives).

Reply:
ahh ok nvm then!